src/main/webapp/WEB-INF/tags/student/profile/studentProfileDiv.tag

```diff
@@ -32,6 +32,16 @@
         Upload/Edit Photo
       </button>
     </div>
+    <br>
+    <br>
+    <div class="">
+      <button id="deletePhoto"
+              class="btn btn-primary"
+              href="${profile.deleteUrl}"
+              type="button">
```

Comment:
All of the above can fit into one line?

Reply:
Actually the rest of the code in this file is formatted like this only. So uniformity should be maintained?